docs/release-notes/8.0.44-35.md

```diff
@@ -0,0 +1,93 @@
+
+### MyRocks
+
+* [PS-9220](https://perconadev.atlassian.net/browse/PS-9220): During slow‑query‑log rotation the server attempts to write to the base file `slow.log` (which isn’t created), raises an error - `errno 2 – No such file or directory`, but the logs are written with numeric suffixes (for example, `slow.log.000087`) that obey the configured size and file‑count limits.
+
+* [PS-9452](https://perconadev.atlassian.net/browse/PS-9452): When the server uses a thread‑pool (thread_handling = pool‑of‑threads) and the variable `thread_pool_high_prio_mode` is set to `transactions`, every new client connection is placed in the normal‑priority queue. The queue is processed only if there is a free thread in the pool.
+
+* [PS-9680](https://perconadev.atlassian.net/browse/PS-9680): Calling `ALTER TABLE … IMPORT TABLESPACE` (or any RocksDB operation that creates a new table) from many threads at the same time can trigger the error “Invalid argument: Global seqno is required, but disabled.”
+Even when `rocksdb_bulk_load` is set to `0`, this error indicates that RocksDB’s global sequence‑number tracking is turned off while the engine still expects it, causing write‑order mismatches that may corrupt the data.
+
+* [PS-9834](https://perconadev.atlassian.net/browse/PS-9834): The Audit Log Filter incorrectly deletes all rotated logs if the `audit_log_filter.max_size` is set > 2 GiB.
+
+* [PS-9840](https://perconadev.atlassian.net/browse/PS-9840): An assertion was triggered because a data dictionary table existed for table T, yet that table was not registered in RocksDB. The storage engine could not locate the corresponding physical representation, leading to the failure.
+
+* [PS-9842](https://perconadev.atlassian.net/browse/PS-9842): An assertion was raised because `rocksdb::MergingIterator::CurrentForward()` asserted that `minHeap_.empty() || minHeap_.top()->type == HeapItem::Type::ITERATOR`. The fix corrected the heap‑state handling, eliminating the SIG 6 termination and stabilized table‑check operations.
+
+* [PS-10067](https://perconadev.atlassian.net/browse/PS-10067): This fix resolves malformed reads and improves the stability of index iteration. The update fixed `Rdb_convert_to_record_key_decoder::skip()` so that it now advances both the key‑slice reader and the unpack‑information reader. By consuming the per‑field metadata (item 5) during iteration with `Rdb_key_field_iterator::next()`, the function correctly handles index fields created via `make_unpack_unknown_varlength()`.
+
+* [PS-10075](https://perconadev.atlassian.net/browse/PS-10075): A checksum‑mismatch failure when executing queries with RocksDB enabled with default settings. The fix corrected the checksum validation logic and ensured that index scans correctly verified and updated record checksums, eliminating the “record checksum mismatch” and “corrupt” messages during table checks.
```

Comment:
An issue could cause a checksum mismatch when reading records. In some cases, the reader would skip per-field metadata but still attempt to read the checksum, resulting in an invalid position. This fix ensures that the checksum is only verified when the reader is correctly positioned, preventing false checksum errors.